### ISSUE TITLE
Abnormal loss in training SSDLite-Mobilenet V2

### ISSUE THREAD

**Issue by MLbeginner2** [qfgaohao/pytorch-ssd#92]
During the training of mobilenet V2, the loss of the first epoch is normal, and the loss value of the subsequent epoch  increases exponentially and there is no downward trend.why is this happening?
![image](https://user-images.githubusercontent.com/45509031/71175728-360c2280-22a3-11ea-88a5-f7a3d7c3f207.png)
